src/guides/v2.3/extension-dev-guide/framework/datetime.md

```diff
@@ -0,0 +1,54 @@
+---
+group: php-developer-guide
+title: DateTime Library
+contributor_name: Rakesh Jesadiya
+contributor_link: https://github.com/RakeshJesadiya
+---
+
+## Overview
+
+This DateTime library provides utilities to work with formatting of date time. Provided methods in this library are described below.
+
+## Usage
+
+|Method|Description|
+|--- |--- |
+| `formatDate` | Format date to internal format |
+| `isEmptyDate` | Check whether sql date is empty |
```

Comment:
This description needs to be rewritten, but I don't know how. `sql` probably  needs to be capitalized. Is the method checking the database or somewhere else?  Maybe the description should be something like "Check whether the date has been stored in the database"